share/man/man7/hier.7

```diff
@@ -97,12 +102,12 @@ boot loader configuration; see
 .Xr loader.conf 5
 configuration files
 .It Pa lua/
-scripts for kernel bootstrapping final stage; see
-.Xr loader_lua 8
+final stage kernel boot
+.Xr loader_lua 8 scripts
```

Comment:
I'd go more for "Scripts for the Lua boot loader." Final stage is a misnomer, and needlessly specific and also confusion since this final stage boot loader can chain boot something else that loads something else... "Kernel" doesn't add much either. We're giving a point to sort out all these nuances. And often times it's the only boot loader...